learning_observer/learning_observer/routes.py

```diff
@@ -62,9 +77,20 @@ def tracemalloc_handler(request):
             aiohttp.web.get('/debug/tracemalloc/', tracemalloc_handler),
         ])
 
-    register_dashboard_api(app)
+    if not settings.pmss_settings.disable_dashboard_routes(types=['server']):
+        register_dashboard_api(app)
+        debug_log("Dashbord routes are enabled")
+    else:
+        debug_log("Dashboard routes are disabled")
+
     register_static_routes(app)
-    register_incoming_event_views(app)
+    
+    if not settings.pmss_settings.disable_extension_routes(types=['server']):
+        register_incoming_event_views(app)
+        debug_log("Extension routes are enabled")
+    else:
+        debug_log("Extension routes are disabled")
+    
```

Comment:
General comment: I could see instances where we want to enable/disable a lot more of the Learning Observer routes on the system.
Not sure that it's needed for the first pass, but would be good to think about and toy around with.